index.html

```diff
@@ -61,6 +60,304 @@ <h1 class="about">
+      </li>
+      <li class="element">
+        <a href="#"><img src="./images/mail.svg" alt="" class="social"></a>
+      </li>
+      <li class="element">
+        <a class="mail" href="#"><img src="./images/github.svg" alt="" class="social"></a>
+      </li>
+      <li class="element">
+        <a class="Linkedin" href="#"><img src="./images/other.svg" alt="" class="social"></a>
+      </li>
+    </ul>
+    <div class="btn d-flex">
+      <div class="action-btn-2">
+        <a href="#">Get my resume</a>
+      </div>
+    </div>
+    <div class="skill-list">
+      <div class="skill-item d-flex">
+        <div class="languages">
+          <p>Languages</p>
+        </div>
+        <div class="btn-container">
+          <div class="normal-arrow-btn">
+            <img src="./images/arrow.png" alt="">
+          </div>
+        </div>
+      </div>
+      <div class="languages-list">
+        <div class="js-frame d-flex">
+          <img src="./images/js-frame.png" alt="">
+          <p class="js-frame-text">
+            JavaScript
+          </p>
+        </div>
+        <div class="html-frame d-flex">
+          <img src="./images/html-frame.png" alt="">
+          <p class="html-frame-text">
+            HTML
+          </p>
+        </div>
+        <div class="css-frame d-flex">
+          <img src="./images/css-frame.png" alt="">
+          <p class="css-frame-text">
+            CSS
+          </p>
+        </div>
+      </div>
+      <div class="skill-item d-flex separator-bot">
+        <div class="frameworks">
+          <p>Frameworks</p>
+        </div>
+        <div class="btn-container">
+          <div class="normal-arrow-btn">
+            <img src="./images/chevron-right.png" alt="">
+          </div>
+        </div>
+      </div>
+      <div class="skill-item d-flex separator-bot">
+        <div class="skills">
+          <p>Skills</p>
+        </div>
+        <div class="btn-container">
+          <div class="normal-arrow-btn">
+            <img src="./images/chevron-right.png" alt="">
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+  <section class="contact">
+    <div class="contact-container d-flex">
+      <div class="contact-text">
+        <h1 class="main-contact">
+          Contact me
+        </h1>
+        <p class="contact-description">
+          If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I'd love to help with it
+        </p>
+      </div>
+        <div class="column d-flex">
+          <input type="text" class="form-control" id="contact" placeholder="Your Name">
+          <input type="email" class="form-control" id="email" placeholder="Your Email">
+          <textarea class="form-control" id="message" placeholder="Your Message"></textarea>
+          <div class="get-in-touch">
+            Get in Touch
+          </div>
```

Comment:
- [ ] Instead `div` tag, you should use the `a` or `button` tag for the "Get in Touch" button, also I can not see the button, please try to fix it with CSS, maybe using the `z-index` attribute.
- [ ] Try to fix the margin/padding for `input` tags in your contact form.

![image](https://user-images.githubusercontent.com/60164239/129303374-eff6a768-8070-4144-9ebe-281855b8650f.png)

Reply:
Can you let me know wich screen width were you using, because padding and margin are looking good for me on my project. Thanks.